python/lsst/validate/drp/calcSrd.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from __future__ import print_function, division
+
+import numpy as np
+import scipy.stats
+
+# To validate these width estimates, we can plot a single random realization (re-evaluate the cell or call the function again to get a new one).
+def calcPA1(gv, magKey):
```

Comment:
A docstring would help here in a few ways:

* While the cognoscenti will know where to look up what PA1 actually refers to, the casual reader won't -- please define it or provide a reference to an external definition;
* I can guess what `magKey` is, but on first read through of the code I have absolutely know idea what `gv` is supposed to be.
* There's a tuple with several fields being returned: please provide advice about how to interpret it.